src/renderer/modules/components/Category.tsx

```diff
@@ -0,0 +1,88 @@
+import { filters, waitForModule } from "../webpack";
+import { Divider, FormItem, FormText } from ".";
+import React from "../webpack/common/react";
+
+const classes = (await waitForModule(
+  filters.byProps("container", "labelRow", "title", "control", "note"),
+  { timeout: 10000 },
```

Comment:
Decrease to `5_000` for consistency with common modules, assuming that doesn't cause issues. Same for all other timeouts.